README.md

```diff
@@ -122,6 +122,28 @@ shipment_quote = Shipcloud::ShipmentQuote.create(
 shipment_quote.price # => 6.2
 ```
 
+### Create a pickup request
+
+To request parcels being picked up, you need to provide the carrier name and the time (earliest and latest) your shipments shall be fetched. You may also provide a list of shipment ids to specify only certain shipments to be included in the pickup request.
```

Comment:
I would rather document the more complete form, specifying the format the shipment list is to be built.